.gitignore

```diff
@@ -8,3 +8,5 @@ node_modules
 
 npm-debug.log*
 pgdata
+__files
+api/test-data/files
```

Comment:
`__files` is where files are uploaded in local dev. We don't want those going into git. (And you might want to remember to clean those out periodically 😁)

`api/test-data/files` is where files are stored for endpoint testing. There is one file there, and this rule will prevent git from capturing any new files that end up there - for example, after running the endpoint tests.